HW2/src/main/java/com/example/hw2/servise/StudentService.java

```diff
@@ -0,0 +1,23 @@
+package com.example.hw2.servise;
+
+import com.example.hw2.entity.Student;
+import com.example.hw2.repository.StudentsRepository;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@Profile("dev")
+public class StudentService {
+
+    private StudentsRepository repository;
+
+    public StudentService(StudentsRepository repository) {
+        this.repository = repository;
+    }
+
+    public List<Student> getAll(){
+        return repository.findAll();
```

Comment:
Prefer do not use this method at all.
It pull whole table from database, in productions most likely it will take few Gb of data from DB and your app will crush (it will be a prod incident).

**Always** use limitation for this kind of selecting.